packages/webapp/components/layouts/FooterNavBarLayout.tsx

```diff
@@ -21,20 +20,20 @@ export default function FooterNavBarLayout({
 }: FooterNavBarLayoutProps): ReactElement {
   const { windowLoaded } = useContext(ProgressiveEnhancementContext);
   const { sidebarRendered } = useSidebarRendered();
-  const { isNewMobileLayout } = useMobileUxExperiment();
   const isTablet = useViewSize(ViewSize.Tablet);
+  const isLaptop = useViewSize(ViewSize.Laptop);
 
   const showNav = useMemo(() => {
     if (!windowLoaded) {
       return false;
     }
 
-    if (isNewMobileLayout) {
+    if (!isLaptop) {
```

Comment:
We can change this line 31 to line 36 with `return isMobile`. The code at line 36 basically checks whether it is a touch device or not.